mig_0001_create_events_targets_tbl.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020, 2021, 2022 Red Hat, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"database/sql"
+
+	mig "github.com/RedHatInsights/insights-operator-utils/migrations"
+	types "github.com/RedHatInsights/insights-results-types"
+)
+
+var mig0001CreateEventTargetsTbl = mig.Migration{
+	StepUp: func(tx *sql.Tx, _ types.DBDriver) error {
+		_, err := tx.Exec(`
+			CREATE TABLE event_targets (
+				id				INTEGER NOT NULL,
+				name          	VARCHAR NOT NULL,
+				metainfo        VARCHAR NOT NULL,
```

Comment:
missing space :)

Reply:
also name and metainfo should be unique (constraint)

Reply:
the space it's like this only on git hub, in the edito is fine

Reply:
there's tab inside the string, which might be problematic later, let's see (still minor thing)

Reply:
right! fixed, thx